website/src/components/articles/article-content-footer.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+
+interface Props {
+  lastUpdated: string;
+  lastAuthorName: string;
+}
+
+export const ArticleContentFooter: FC<Props> = ({
```

Comment:
For consistency and readability purposes, I prefer FunctionComponent. I'm generally open to discuss that, but for now I would prefer consistency. Thank you.

Reply:
Made the change for consistency, but I personally prefer FC. In my opinion it makes the code more readable, since there's less clutter.

Reply:
We can discuss conventions of course. FC is fine with me, but then consistent. If you would like to change, then go ahead, but please do in a separate branch and for the entire codebase. Thanks!